launchers/configure.sh

```diff
@@ -71,25 +71,12 @@ else
   readonly ICO_TARGET_DIR=$ICO_TARGET_DIR
 fi
 
-if [ "x$SNAPSHOT" == "x" ] ; then
```

Comment:
The comparison on line 74 is implemented different then on line 92.
My shell knowledge is more on a 'monkey see - monkey do' level. So I am not sure if both of them behave exactly the same.
Never the less I would suggest to stick with only one approach to keep things consistent